### ISSUE TITLE
Would be nice if redirects could point to scenarios on staging server

### ISSUE THREAD

**Issue by AlexanderWirtz** [quintel/etcentral#139]
We have created redirects for the IABR and Rli scenarios so that we can use pretty looking links in our reports. These are specified in etcentral/config/initializers/redirects.rb

It would be nice if these redirects could also point at scenarios on the staging server. We have a slightly prettier scenario on the staging server for IABR that used some new `adjust_scaling` input statements in `etsource` that have not been deployed tot Live yet.

In short, until we cherry pick or deploy these new input statements to Live, I would like:
`'/iabr/routekaartscenario-2035'` to point to: `https://beta-pro.energytransitionmodel.com/scenarios/519784`

and

`'/iabr/routekaartscenario-2025'` to point to: `https://beta-pro.energytransitionmodel.com/scenarios/519786`

notifying @dennisschoenmakers

**Comment by AlexanderWirtz** [comment]
@DennisSchoenmakers what is the status of this issue?

**Comment by dennisschoenmakers** [comment]
Fixed with #140 .
